Description: disable aborting cxx tests
 These cxx tests die from sigabrt, disabling for now.
Author: Ross Vandegrift <ross@kallisti.us>
Origin: other
Forwarded: no
Last-Update: 2017-04-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/tests/eo_cxx/eo_cxx_suite.cc
+++ b/src/tests/eo_cxx/eo_cxx_suite.cc
@@ -15,6 +15,9 @@
 {
    int failed_count;
 
+   fprintf(stderr, "Skipping %s\n", argv[0]);
+   return(77);
+
    if (!_efl_test_option_disp(argc, argv, etc))
      return 0;
 
--- a/src/tests/ecore_cxx/ecore_cxx_suite.cc
+++ b/src/tests/ecore_cxx/ecore_cxx_suite.cc
@@ -15,6 +15,9 @@
 {
    int failed_count;
 
+   fprintf(stderr, "Skipping %s\n", argv[0]);
+   return(77);
+
    if (!_efl_test_option_disp(argc, argv, etc))
      return 0;
 
--- a/src/tests/eldbus_cxx/eldbus_cxx_suite.cc
+++ b/src/tests/eldbus_cxx/eldbus_cxx_suite.cc
@@ -15,6 +15,9 @@
 {
    int failed_count;
 
+   fprintf(stderr, "Skipping %s\n", argv[0]);
+   return(77);
+
    if (!_efl_test_option_disp(argc, argv, etc))
      return 0;
 
